packages/TextEditor/stories/examples/controlled.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import TextEditor from "../../src";
+
+export default function TextEditorSimple() {
+  const API_KEY = "t9jqh1aieshznn654ruvg53914pbzyqcni0bhzbcl1iznj8v";
```

Comment:
nit: pull this out into a config file

Reply:
Don't know  how are we going to manage this, if this is part of this package or should be set on a CI. right now will stay like that. I comment about this on the PR summary.

Reply:
I didnt mean for secrecy, i meant so it is doesnt have to be duplicated each time a story is created. not that important.